README.md

```diff
@@ -464,6 +464,25 @@ withPipeline(type, product, component) {
 
 ```
 
+## Import Service Bus, Topic, Queue and Subscription modules from template deployment to native Terraform resources
+
+#### Usage
+
+The existing Service Bus and its related modules use Azure Template Deployment to deploy the resources. Terraform has released native resources to managed these resources. Upgraded versions of these modules are available in a separate branch.
+
+To consume the new modules, existing resources must be imported to the new module structure. This can be achieved by updating the branch reference as shown in the Git Commit below and using the `importServiceBusModules()` method in Application and Infrastructure pipelines. It is advisable to run the pipeline with this method call once. Subsequent runs will create an empty stage, which can be avoided.
+
+Github Commit: https://github.com/hmcts/rd-shared-infrastructure/commit/5f1e464f246f8ed343951f044f1477667c248908#
+
+Build Console: https://sandbox-build.platform.hmcts.net/job/HMCTS_Sandbox_RD/job/rd-shared-infrastructure/job/sandbox/135/console
+
+```groovy
+
+withPipeline(type, product, component) {
+    importServiceBusModules()
```

Comment:
was wondering if we can make this generic - like `import terraformModules(Importer.SERVICEBUS,....)` So that this can be extended to other imports we may need to do in future like postgresql module. Having set it up as a framework might be helpful in long run. Needless to say, this is a very valuable contribution and thanks for that 👏

Reply:
@adusumillipraveen, good thought. We can investigate how to achieve this. At the moment the resource type and name is checked before making a decision. In the future, we can also check for the name applicable for postgres